ynput/ayon-backend
# Settings: copy project settings when promoting a bundle

When promoting a bundle from staging to production. All settings (site and project) should be copied during the process.

## Testing steps 

- Create a bundle and set it as staging
- Edit studio and project staging settings 
- Hit the rocket icon and confirm the dialog
- The bundle should be now be both in "production" and "staging" and the settings should be the same
